book/scheduling.md

```diff
@@ -1135,11 +1121,15 @@ class TaskRunner:
 
     def run(self):
         while True:
-            # ...
+            self.lock.acquire(blocking=True)
+            needs_quit = self.needs_quit
```

Comment:
You'll need to introduce the variable and the code that sets it then. Is that really worth it?

Reply:
Ok, good point.